#!/usr/bin/env python

from dynamic_reconfigure.parameter_generator_catkin import *
PACKAGE = "laser_filters"

gen = ParameterGenerator()

##### gen.add("name", paramtype, level, description, default, min, max)

gen.add("inscribed_radius", double_t, 0, "radius of circle where the points inside are removed", 2.5, -20, 20)
gen.add("switch_", bool_t, 0, "Switches filter on or off", True)

exit(gen.generate(PACKAGE, "laser_filters", "FootprintFilter"))